include/zephyr/bluetooth/services/ans.h

```diff
@@ -0,0 +1,135 @@
+/*
+ * Copyright (c) 2025 Sean Kyer
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef ZEPHYR_INCLUDE_BLUETOOTH_SERVICES_ANS_H_
+#define ZEPHYR_INCLUDE_BLUETOOTH_SERVICES_ANS_H_
+
+/**
+ * @brief Alert Notification Service (ANS)
+ * @defgroup bt_ans Alert Notification Service (ANS)
+ *
+ * @since 4.3
+ * @version 0.1.0
+ *
+ * @ingroup bluetooth
+ * @{
+ */
+
+#include <stdint.h>
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @brief Command not supported error code
+ */
+#define BT_ANS_ERR_CMD_NOT_SUP 0xa0
+
+/**
+ * @brief ANS max text string size
+ *
+ * This is the max string length to be saved in a New Alert. Text longer than the max is truncated.
+ *
+ * section 3.165 of
+ * https://btprodspecificationrefs.blob.core.windows.net/gatt-specification-supplement/GATT_Specification_Supplement.pdf
+ *
+ */
+#define BT_ANS_MAX_TEXT_STR_SIZE 18
+
+/**
+ * @brief ANS Category ID Enum
+ *
+ * Enumeration for whether the category is supported.
+ */
+enum bt_ans_category {
+	BT_ANS_CAT_SIMPLE_ALERT,    /**< Simple alerts (general notifications). */
```

Comment:
I'm not aware of any official guidelines on this, but I have a personal preference to use the same prefix for the enum and the values, i.e. either rename as


```suggestion
enum bt_ans_category {
	BT_ANS_CATEGORY_SIMPLE_ALERT,    /**< Simple alerts (general notifications). */
```

or

```suggestion
enum bt_ans_cat {
	BT_ANS_CAT_SIMPLE_ALERT,    /**< Simple alerts (general notifications). */
```